special-cat.md

```diff
@@ -0,0 +1 @@
+A special cat is a sweet and delicious pink squishy cat made out of marshmallows. Its meows are made of candy floss and its coat is spun sugar. If you ask nicely, he will make you a nutella crepe.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.